File: soknad/src/main/java/no/nav/syfo/kafka/soknad/dto/SporsmalDTO.java
package no.nav.syfo.kafka.soknad.dto;

import lombok.Builder;
import lombok.Value;

import java.util.List;

@Value
@Builder(toBuilder = true)
public class SporsmalDTO {
    String id;
    String tag;
    String sporsmalstekst;
    String undertekst;
    String svartype;
    String min;
    String max;
    String kriterieForVisningAvUndersporsmal;
    List<SvarDTO> svar;
    List<SporsmalDTO> undersporsmal;
}

File: soknad/src/main/kotlin/no/nav/syfo/kafka/soknad/dto/SporsmalDTO.kt
package no.nav.syfo.kafka.soknad.dto

data class SporsmalDTO(
        val id: String? = null,
        val tag: String? = null,
        val sporsmalstekst: String? = null,
        val undertekst: String? = null,
        val svartype: String? = null,
        val min: String? = null,
        val max: String? = null,
        val kriterieForVisningAvUndersporsmal: String? = null,
        val svar: List<SvarDTO>? = null,
        val undersporsmal: List<SporsmalDTO>? = null
)
